Tests iOS/EditTagsTests.swift

```diff
@@ -28,44 +30,50 @@ class EditTagsTests: XCTestCase {
         try server.start()
     }
 
-    override func tearDownWithError() throws {
+    @MainActor
+    override func tearDown() async throws {
         try server.stop()
         app.terminate()
+        await snowplowMicro.assertNoBadEvents()
```

Comment:
@cyndichin lets use https://github.com/Pocket/pocket-ios/blob/develop/Tests%20iOS/Support/SnowplowMicro.swift#L296-L298 instead

Reply:
ah nice catch! I realize I was also adding that to individual tests, but can prob remove it.

Reply:
updated!